storybookjs/storybook
# Preview: Don't hide the story while preparing

Issue: #16847, #16798

## What I did

Removed the code to hide the story while not "showing" (see [this comment](https://github.com/storybookjs/storybook/issues/16847#issuecomment-983129179)). 

Note the original code:

```js
:not(.sb-show-main) > .sb-main
```

Did not apply to anything, so I didn't put it back. There is `.sb-main-padded`, etc for the different layouts, but there is no `sb-main-x` class for when you have no layout (the default).

## How to test

1. Open a preview, remove `sb-show-main`, add `sb-show-preparing-story` or `sb-show-preparing-docs` to the `body`. Check the story is not visible.
2. Run `examples/vue-kitchen-sink`, visit http://localhost:9009/?path=/docs/addon-actions--action-only (for #16798)
